.gitignore

```diff
@@ -0,0 +1 @@
+.vscode/
```

Comment:
We already have a .gitignore inside folder, but it's not having the impact. Maybe you can delete the [current gitignore file](https://github.com/AOT-Technologies/forms-flow-ai-extensions/blob/master/camunda-formio-tasklist-vue/.gitignore) and modify the contents in file location you have shared now.